src/pinns_pde_solve.jl

```diff
@@ -532,17 +532,17 @@ Base.Broadcast.broadcasted(::typeof(get_u()), cord, θ, phi) = get_u()(cord, θ,
 
 # the method to calculate the derivative
 function get_numeric_derivative()
-    _epsilon = 1/(2*cbrt(eps(Float32)))
+    _epsilon = (2*cbrt(eps(Float32)))
     derivative = (phi,u,x,εs,order,θ) ->
     begin
         ε = εs[order]
         ε = adapt(typeof(θ),ε)
         x = adapt(typeof(θ),x)
         if order > 1
             return (derivative(phi,u,x .+ ε,εs,order-1,θ)
-                  - derivative(phi,u,x .- ε,εs,order-1,θ))*_epsilon
+                  .- derivative(phi,u,x .- ε,εs,order-1,θ)) /_epsilon
         else
-            return (u(x .+ ε,θ,phi) - u(x .- ε,θ,phi))*_epsilon
+            return (u(x .+ ε,θ,phi) .- u(x .- ε,θ,phi)) /_epsilon
```

Comment:
./?

Reply:
you might as well fuse it.

Reply:
done

Reply:
@ChrisRackauckas `fastchain` doesn't work with gpu?

Reply:
it does. There's specific tests for it.